### PR TITLE
Add some additional package props

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Based on https://github.com/Azure/azure-webjobs-sdk/tree/dev/src/Microsoft.Azure.WebJobs.Extensions.Storage

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
     <Authors>Microsoft</Authors>
     <Product>SQL Binding Extension</Product>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <DebugSymbols>true</DebugSymbols>
     <IncludeSymbols>false</IncludeSymbols>
     <DebugType>embedded</DebugType>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Sql</PackageId>
+    <PackageTags>Microsoft Azure WebJobs AzureFunctions SQL</PackageTags>
```
Comment (chlafreniere) [inline src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:15]: Is there an "AzureSQL" package tag that we should add as well?

**Review by chlafreniere — APPROVED** [review]

File: src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj (hunk 1 above)
Comment (Charles-Gagnon) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:15]: I'm not seeing anything like that when I search https://www.nuget.org/packages?q=azure+sql

Is AzureSQL (all one word) something you think people would actually be searching? As long as you have azure or sql in it then it'll be fine and that seems to me to be the more common phrases to search

**Review by lucyzhang929 — APPROVED** [review]

File: src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj (hunk 1 above)
Comment (chlafreniere) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:15]: I have no data to support "AzureSQL". Just wondering if there was another tag beyond "SQL", that's all 😄 . Totally fine.

Comment (Charles-Gagnon) [reply src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj]: Added it, not like it's going to cause problems and might be useful to someone

**Pull request merged**